apps/api/v2/src/ee/event-types/event-types_2024_06_14/services/output-event-types.service.ts

```diff
@@ -87,24 +109,42 @@ export class OutputEventTypesService_2024_06_14 {
       scheduleId,
       onlyShowFirstAvailableSlot,
       offsetStart,
+      requiresBookerEmailVerification,
+      hideCalendarNotes,
+      seatsShowAttendees,
+      useEventTypeDestinationCalendarEmail,
     } = databaseEventType;
 
     const locations = this.transformLocations(databaseEventType.locations);
+    const customName = databaseEventType?.eventName ?? undefined;
     const bookingFields = databaseEventType.bookingFields
       ? this.transformBookingFields(BookingFieldsSchema.parse(databaseEventType.bookingFields))
       : [];
     const recurrence = this.transformRecurringEvent(databaseEventType.recurringEvent);
     const metadata = this.transformMetadata(databaseEventType.metadata) || {};
-    const users = this.transformUsers(databaseEventType.users);
+    const users = this.transformUsers(databaseEventType.users || []);
     const bookingLimitsCount = this.transformIntervalLimits(databaseEventType.bookingLimits);
     const bookingLimitsDuration = this.transformIntervalLimits(databaseEventType.durationLimits);
+    const color = this.transformEventTypeColor(databaseEventType.eventTypeColor);
+    const bookerLayouts = this.transformBookerLayouts(
+      metadata.bookerLayouts as unknown as BookerLayoutsTransformedSchema
+    );
+    const confirmationPolicy = this.transformRequiresConfirmation(
+      !!databaseEventType.requiresConfirmation,
+      !!databaseEventType.requiresConfirmationWillBlockSlot,
+      metadata.requiresConfirmationThreshold as NoticeThresholdTransformedSchema
+    );
+    delete metadata["bookerLayouts"];
+    delete metadata["requiresConfirmationThreshold"];
```

Comment:
Why are we deleting these?

Reply:
the reason we delete these are becase these fields are stored in metada in our db, but we have separate fields for them in our API input, so sending these properties in the metadata is a bit redundant since we are already transforming and sending these in `bookerLayouts` and `confirmationPolicy.noticeThreshold`

![Screenshot 2024-09-20 at 7 48 58 PM](https://github.com/user-attachments/assets/20ee83f4-23eb-43bc-a08e-4d176c001599)

![Screenshot 2024-09-20 at 7 50 37 PM](https://github.com/user-attachments/assets/5814ab91-422f-4265-945a-f7a827eae2c7)